### PR TITLE
Five stories

### PR DESCRIPTION
**Pull request by terrifricker**
## Description
Changes made:
1. A file data-template.json holds a template for the stories' json.
2. Three stories were added to data.json to bring us to a total of five stories.
3. Added code to app.json to hide the previous story when a Story Title button is clicked. (bug fix)

## Acceptance Criteria

- [ ] Clicking a Story Title button brings up the form requesting words for the blanks in the story
- [ ] Filling in the blanks and clicking the Submit button brings up a story with the words inserted
- [ ] The stories make sense and have correct grammar
- [ ] Clicking Play Again button clears the screen
- [ ] Clicking a Story Title button will clear previous story and bring up new form

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
| ✓  | :bug: Bug fix              |
| ✓  | :sparkles: New feature     |
| ✓  | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
data.json
app.js
adds file data-template.json

### Before

Opening play.html with just two Story Title buttons
Screen Shot 2023-03-31 at 10 36 41 AM

### After

Opening play.html with five Story Title buttons
Screen Shot 2023-03-31 at 10 20 37 AM

Clicking on story "Animal Friends" (no change from previous UI)
Screen Shot 2023-03-31 at 10 21 06 AM

After entering words and clicking Submit button (no change from previous UI)
Screen Shot 2023-03-31 at 10 23 13 AM

### CODE REVIEW

File: app.js
Comment (terrifricker) [inline app.js:1]: Ack!  I had formatting changes, too!  I forgot to do them separately, too!

The only **real** change here is the addition of lines 29, 30, and 31
```
			// hide form and story
			wordForm.classList.add('hidden');
			story.classList.add('hidden');
```

File: data/data-template.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "title": "Template",
+    "id": 0,
```
Comment (terrifricker) [inline data/data-template.json:3]: This is for future javascript to grab a random story

**Review by mmsaki — APPROVED** [review]
Nice! We have more stories 👍🏾

**Comment by mmsaki** [comment]
I don't see any bug fixes bot sure why this has a bug label

**Comment by terrifricker** [comment]
> I don't see any bug fixes bot sure why this has a bug label

I had a bug in my app.js in that my code didn't hide the previous story when a new Story Title button was clicked.  But I never did make an issue for that in particular, just fixed it here.

**Pull request merged**